src/Stratis.Bitcoin.Features.Dashboard/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+using System.IO;
```

Comment:
I don't think that all these usings are required? Can you download and install Productivity Power Tools for Visual Studio? There is a setting where you can remove and sort usings on save, pretty cool!